src/Compilers/Core/Portable/InternalUtilities/FatalError.cs

```diff
@@ -20,8 +20,12 @@ namespace Microsoft.CodeAnalysis.ErrorReporting
 {
     internal static class FatalError
     {
+        // A delegate type for our NonFatalHandler; this is necessary since this file must build on .NET 2.0 where
+        // multi-parameter Action<...> hadn't been invented yet.
```

Comment:
Nit: any reason this isn't a doc comment?

Reply:
Fixed.